src/AwesomeAlert.js

```diff
@@ -16,7 +15,9 @@ import PropTypes from 'prop-types';
 
 import styles from './styles';
 
-const HwBackHandler = BackHandler || BackAndroid;
+// BackAndroid was removed from react-native
+// const HwBackHandler = BackHandler || BackAndroid;
```

Comment:
```suggestion
// BackAndroid was removed from react-native
// const HwBackHandler = BackHandler || BackAndroid;
```
you can safely remove these comments, these probably won't be used in the future